volumes.h

```diff
@@ -40,12 +60,15 @@ struct pv_volume {
 	int file_fd;
 	struct pv_platform *plat;
 	struct dl_list list; // pv_volume
+	struct pv_disk *disk;
 };
 
 void pv_volume_free(struct pv_volume *v);
 
-struct pv_volume* pv_volume_add(struct pv_state *s, char *name);
+struct pv_disk* pv_disk_add(struct pv_state *s);
+void pv_disks_empty(struct pv_state *s);
 
+struct pv_volume* pv_volume_add(struct pv_state *s, char *name, char *disk);
```

Comment:
we could make the diff smaller and also the function not as obscure in normal case if we add a pv_volume_add_wiith_disk() and have pv_volume_add call that function with NULL .... then we only need to use pv_volume_add_wiith_disk really where we need it...

Reply:
Addressed, thanks